app/jobs/create_account_fcrepo_job.rb

```diff
@@ -0,0 +1,6 @@
+# This job is the same as its parent, but chains to the next step
+class CreateAccountFcrepoJob < CreateFcrepoEndpointJob
```

Comment:
Right, I gather from these comments that this approach has been demonstrated adequately and is broadly acceptable.  I would agree that only one version of that class is necessary (right now).